### PR TITLE
Grant MANAGE_USERS to pixelmigrate

### PR DESCRIPTION
**Pull request by ix5**
Needed because VM will shut down on recent Pie builds.

```
01-11 08:33:01.684  6118  6118 W PackageManager: Privileged permission android.permission.MANAGE_USERS for package com.google.android.apps.pixelmigrate - not in privapp-permissions whitelist
01-11 08:33:01.692  6118  6118 V UserDataPreparer: Found /data/user_de/0 with serial number 0
01-11 08:33:01.692  6118  6118 V UserDataPreparer: Found /data/user/0 with serial number 0
01-11 08:33:01.692  6118  6118 V UserDataPreparer: Found /data/system_de/0 with serial number 0
01-11 08:33:01.693  6118  6118 E System  : ******************************************
01-11 08:33:01.693  6118  6118 E System  : ************ Failure starting system services
01-11 08:33:01.693  6118  6118 E System  : java.lang.IllegalStateException: Signature|privileged permissions not in privapp-permissions whitelist: {com.google.android.apps.pixelmigrate: android.permission.MANAGE_USERS}
01-11 08:33:01.693  6118  6118 E System  : 	at com.android.server.pm.permission.PermissionManagerService.systemReady(PermissionManagerService.java:2005)
01-11 08:33:01.693  6118  6118 E System  : 	at com.android.server.pm.permission.PermissionManagerService.access$100(PermissionManagerService.java:89)
01-11 08:33:01.693  6118  6118 E System  : 	at com.android.server.pm.permission.PermissionManagerService$PermissionManagerInternalImpl.systemReady(PermissionManagerService.java:2052)
01-11 08:33:01.693  6118  6118 E System  : 	at com.android.server.pm.PackageManagerService.systemReady(PackageManagerService.java:21210)
01-11 08:33:01.693  6118  6118 E System  : 	at com.android.server.SystemServer.startOtherServices(SystemServer.java:1694)
01-11 08:33:01.693  6118  6118 E System  : 	at com.android.server.SystemServer.run(SystemServer.java:431)
01-11 08:33:01.693  6118  6118 E System  : 	at com.android.server.SystemServer.main(SystemServer.java:294)
01-11 08:33:01.693  6118  6118 E System  : 	at java.lang.reflect.Method.invoke(Native Method)
01-11 08:33:01.693  6118  6118 E System  : 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
01-11 08:33:01.693  6118  6118 E System  : 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:838)
01-11 08:33:01.694  6118  6118 E Zygote  : System zygote died with exception
01-11 08:33:01.694  6118  6118 E Zygote  : java.lang.IllegalStateException: Signature|privileged permissions not in privapp-permissions whitelist: {com.google.android.apps.pixelmigrate: android.permission.MANAGE_USERS}
```

### CODE REVIEW

File: etc/permissions/privapp-permissions-google.xml [hunk 1]
```diff
@@ -120,6 +120,7 @@ It allows additional grants on top of privapp-permissions-platform.xml
         <permission name="android.permission.MANAGE_USB"/>
         <permission name="android.permission.SUBSTITUTE_NOTIFICATION_APP_NAME"/>
         <permission name="android.permission.WRITE_SECURE_SETTINGS"/>
+        <permission name="android.permission.MANAGE_USERS"/>
```
Comment (luk1337) [inline etc/permissions/privapp-permissions-google.xml]: perms should be alphabetically ordered therefore you should put it below MANAGE_USB perm.

Comment (ix5) [reply etc/permissions/privapp-permissions-google.xml]: Fixed

Comment (ix5) [reply etc/permissions/privapp-permissions-google.xml]: Could you also put a notice into the HOWTO section?

**Comment by luk1337** [comment]
can you squash these?

**Comment by ix5** [comment]
Sure, but isn't there a button in the github interface that does that?

**Comment by ix5** [comment]
Squashed.

**Comment by NicholasBuse** [comment]
Thanks!

**Comment by ix5** [comment]
Thanks to you as well!

**Pull request merged**